omero/developers/Web/CreateApp.txt

```diff
@@ -33,22 +33,21 @@ as long as it can be imported by OMERO.web.
 Creating an app
 ---------------
 
-We suggest you use GitHub (as we do) since it is much easier for us to
+We suggest you use `GitHub <https://github.com/>`_ (as we do) since it is much easier for us to
 help you with any problems you have if we can see your code. The steps below
 describe how to create a stand-alone git repository for your app, similar to
 `webtagging <https://github.com/dpwrussell/webtagging>`_.
-If you do not want to use GitHub, simply ignore the GitHub steps below.
+If you do not want to use GitHub, simply ignore the steps related to GitHub.
 
-The steps below describe setting up a new app. You should choose an
-appropriate name for your app and use it in place of <your-app> in the
-examples below:
+The steps below describe how to set up a new app. You should choose an
+appropriate name for your app and use it in place of <your-app>:
 
 
 Add your app to your PYTHONPATH
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Your app needs to be within a directory that is on your :envvar:`PYTHONPATH`.
-I usually create a new container for a new app, and add it to my
+We usually create a new container for a new app, and add it to my
```

Comment:
'add it to *my*' reads strangely now this is 'we' rather than 'I' - would 'the PYTHONPATH' work? Otherwise, 'we recommend creating a new container for a new app and adding it to your PYTHONPATH'